docker-compose.yml

```diff
@@ -2,30 +2,30 @@ version: '3.4'
 
 services:
   nginx:
-    image: minkov/nginx:latest
+    build: .
     restart: unless-stopped
     ports:
       - "80:80"
       - "443:443"
     container_name: nginx_container
     restart: on-failure
     volumes:
-      - ./nginx.conf:/etc/nginx/nginx.conf
-      - ./sites-enabled:/etc/nginx/sites-enabled
-      - ./modules-enabled:/etc/nginx/modules-enabled
-      - ./conf.d:/etc/nginx/conf.d
-      - ./lua:/etc/nginx/lua
+      - ./local/nginx.conf:/etc/nginx/nginx.conf
+      - ./local/sites-enabled:/etc/nginx/sites-enabled
+      - ./local/modules-enabled:/etc/nginx/modules-enabled
+      - ./local/conf.d:/etc/nginx/conf.d
+      - ./local/lua:/etc/nginx/lua
       - ./logs:/var/log/nginx
-      - ./shared:/etc/nginx/shared
+      - ./local/shared:/etc/nginx/shared
       - ./data/certbot/conf:/etc/letsencrypt
       - ./data/certbot/www:/var/www/certbot
     extra_hosts:
-      - "localnode:172.18.0.1"
+      - "localnode:192.168.14.138"
```

Comment:
What is this IP?

Reply:
A local IP on my machine. Leave it as is